pandas/core/groupby/groupby.py

```diff
@@ -581,6 +582,17 @@ def wrapper(*args, **kwargs):
             if 'axis' not in kwargs_with_axis or \
                kwargs_with_axis['axis'] is None:
                 kwargs_with_axis['axis'] = self.axis
+            if (name == 'hist' and
+                    kwargs.get('equal_bins', False) is True):
```

Comment:
Do we need `is True` here or can we use the implicit truthiness of objects?

Reply:
It's just a bit safer this way, as it guards against some non-boolean statements by the user, such as `equal_bins=None`. But your suggestion should be fine if we ignore bad inputs from user.